internal/provider/data_source_http_test.go

```diff
@@ -216,6 +222,59 @@ func TestDataSource_UpgradeFromVersion2_2_0(t *testing.T) {
 	})
 }
 
+func TestDataSource_Provisioner(t *testing.T) {
+	t.Parallel()
+
+	svr := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		// A Content-Type that does not raise a warning in the Read function must be set in
+		// order to prevent test failure under TF 0.14.x as warnings result in no output
+		// being written which causes the local-exec command to fail with "Error:
+		// local-exec provisioner command must be a non-empty string".
+		// See https://github.com/hashicorp/terraform-provider-http/pull/74
+		w.Header().Set("Content-Type", "text/plain")
+		w.WriteHeader(http.StatusOK)
+	}))
+	defer svr.Close()
+
+	resource.UnitTest(t, resource.TestCase{
```

Comment:
Nit: Recommend switching this to `Test()` or `ParallelTest()` since this requires network connectivity to download an external provider 👍 

```suggestion
	resource.Test(t, resource.TestCase{
```

Reply:
Updated.